src/lib.rs

```diff
@@ -167,3 +167,55 @@ mod parsing {
         }
     }
 }
+
+#[cfg(feature = "parsing")]
+pub mod parsing {
+    //! This module contains a set of exported nom parsers which can be used to
+    //! build nom parsers for custom grammars when used alongside the `syn_nom`
+    //! crate.
+    //!
+    //! `syn` uses its own custom fork of `nom`, `syn_nom`, which is smaller, and
+    //! thus improves build speeds. This should be used instead of `nom` when
+    //! building parsers using these parsers.
+
+    pub use macro_input::parsing::macro_input;
```

Comment:
Let's leave this out, this one is specific to the macros 1.1 custom derive use case and isn't something people will need for other procedural macros.